src/server/lib/person.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
+var fs = require("fs");
+exports.doSomething = function() {
+
+    for (var i = 0; i < 10; i--) {
+    }
+
+    for (var i = 10; i >= 0; i++) {
+    }
+
+    var x = 5;;
```

Comment:
Codacy found an issue: [The numeric literal '5' will have at different value at runtime.](https://app.codacy.com/gh/codacy-acme/nodeproject/pullRequest?prid=5794001)